docs/setting_zone_region.md

```diff
@@ -16,19 +18,25 @@ docker:
 
```

Comment:
Is having both region and zone here okay?
Also please consider adding a Tested field to the description if you have tested the config here.

Reply:
yes, it's ok for Dataflow flags. It used to be ok even for pipelines API but it has recently changed as 'regions' is a v2 parameter and 'zones' is a v1/v2 parameter. So, with v1 only 'zones' is actually taken into account and the error message was added recently about this.
Also added tested :)